app/assets/v2/js/ptokens/ptokens.js

```diff
@@ -0,0 +1,88 @@
+const TX_STATUS_PENDING = 'pending';
+const TX_STATUS_SUCCESS = 'success';
+const TX_STATUS_ERROR = 'error';
+const TX_STATUS_UNKNOWN = 'unknown';
+const TX_STATUS_DROPPED = 'dropped';
+const redemption_states = [ 'request', 'accepted', 'denied', 'completed' ];
+
```

Comment:
Good way to do this 👍 Do we need a pending redemption status here?

Reply:
No, i just added in case is required, the fronted doens;t have to search in the python code the status.